### PR TITLE
chore(whole site): Remove eg and ie

### PR DESCRIPTION
**Pull request by austin-schaefer**
The usage dictionary has long asked that we not use `e.g.` and `i.e.`. This PR removes all instances of both that I could find outside legal/licenses docs. It also expands the guidance about e.g. and i.e. in the usage dictionary.

More info in usage dictionary: https://docs.newrelic.com/docs/style-guide/quick-reference/usage-dictionary/#eg

It also (why not) removes instances across the site where we have double spaces between sentences.

### CODE REVIEW

**Comment by rhetoric101** [comment]
I have reviewed about 1/2 of these changes, but my local instance starting failing to load pages. I'll try again tommorrow.

File: src/content/docs/infrastructure/host-integrations/installation/container-auto-discovery-host-integrations.mdx [hunk 1]
```diff
@@ -359,6 +359,6 @@ For example, a container that exposes the 80 and 443 private ports via the publi
 
 ### Mapping ports by protocol name
 
-In addition to the indexing from the previous section, ports can be also referred by its protocol name (e.g. `tcp`, `udp`, ...). For example, `${discovery.ports.tcp}` , `${discovery.ports.udp}`, etc...
+In addition to the indexing from the previous section, ports can be also referred by its protocol name (such as `tcp`, `udp`, etc). For example, `${discovery.ports.tcp}`, `${discovery.ports.udp}`.
```
Comment (rhetoric101) [inline src/content/docs/infrastructure/host-integrations/installation/container-auto-discovery-host-integrations.mdx:362]: @austin-schaefer, I think you need a period after the "etc" before the closing parenthesis.

File: src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-logs/infrastructure-agent-logging-behavior.mdx [hunk 2]
```diff
@@ -84,7 +84,7 @@ The infrastructure agent does not provide any native log rotation or compression
     * `copytruncate`: Indicates that the log file is truncated but not deleted when it is rotated. **This configuration option is mandatory**, otherwise the log file will be deleted and won’t be recreated.
     * `compress`: Compresses (usually in Gzip format) the rotated log files.
     * `daily`: The agent rotates logs daily.
-    * `dateext`: Appends a date (by default, in the format YYYYMMDD) to the rotated log file (e.g. `newrelic-infra.log-20190708.gz`)
+    * `dateext`: Appends a date (by default, in the format YYYYMMDD) to the rotated log file (example: `newrelic-infra.log-20190708.gz`).
```
Comment (rhetoric101) [inline src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-logs/infrastructure-agent-logging-behavior.mdx:87]: @austin-schaefer, how about sticking with the formula of "for example," here?

File: src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx [hunk 3]
```diff
@@ -148,7 +148,7 @@ and one pod for any other component that you have enabled.
 
     To install using Helm 2:
 
-    1. Ensure that Helm 2, including their cluster-side components (i.e. Tiller) are properly installed and configured.
+    1. Ensure that Helm 2, including their cluster-side components (such as. Tiller) are properly installed and configured.
```
Comment (rhetoric101) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx:151]: @austin-schaefer, I think there is a stray period after "as" here.

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx [hunk 4]
```diff
@@ -230,31 +227,25 @@ During this phase, you will introduce incident management teams and other stakeh
...
-
-- Are there application / system problems for engineering to review?
-- Are there poorly constructed policies that need to be fixed?
-- Are there instrumentation gaps?
+1. Review AQM Dashboard and KPI Trending: Here you and the stakeholders will look at the AQM KPIs and identify their week over week trends.  The team should identify areas where KPIs are not improving and develop strategies to drive improvement.
+2. Identify Achievements, Challenges, and Opportunities: Here you and the stakeholders will map the current state of alert quality to business impact, identifying areas where improvement has resulted in better business outcomes and areas where problems are impacting business outcomes.
+3. Incident Policy Review: Using the AQM dashboard, you and the stakeholders will identify the noisiest incident policies.  Once identified, those policies should be evaluated as detailed in step 4 below.
+4. Alert Policy Recommendations: In this step, you and the stakeholders will review the noisiest policies using the following criteria:
+    - Do the alerts have any business impact?
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx:234]: @austin-schaefer, should we remove the blank line separating this child bullet from its sibling? This extra line doesn't seem like the pattern we've been using.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx [hunk 5]
```diff
@@ -101,7 +101,7 @@ Similarly `clamp_min(duration, 1)` will not return any value lower than 1. The f
 
 ### `pow`
 
-`pow(n, m)` computes _n_ raised to the power _m_. (I.e. `n * n * ... * n,` with _m_ copies of _n_)
+`pow(n, m)` computes _n_ raised to the power _m_. (for example, `n * n * ... * n,` with _m_ copies of _n_)
```
Comment (rhetoric101) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx:104]: @austin-schaefer, this one looks good. On line 72, there's a "For example" that could use a following comma.

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx [hunk 6]
```diff
@@ -1310,7 +1310,7 @@ In this section we explain NRQL functions, both [aggregator functions](#aggregat
 You can use aggregator functions to filter and aggregate data. Some tips for using these:
 
 * See New Relic University tutorials for [Filter queries](http://newrelic.wistia.com/medias/8k4xbxlfwj), [Apdex queries](http://newrelic.wistia.com/medias/46dvx1mkm0), and [Percentile queries](http://newrelic.wistia.com/medias/j3htrz66l0). Or, go to the full online course [Writing NRQL queries](https://learn.newrelic.com/writing-nrql-queries).
-* If you're using an aggregator function multiple times in the same query (e.g., `SELECT median(one_metric), median(another_metric)`), it can cause problems in displaying results. To solve this, use the [`AS` function](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-as). For example: 
+* If you're using an aggregator function multiple times in the same query (for example `SELECT median(one_metric), median(another_metric)`), it can cause problems in displaying results. To solve this, use the [`AS` function](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions/#sel-as). For example:
```
Comment (rhetoric101) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx:1313]: @austin-schaefer, how about adding a comma after "for example"?

File: src/content/docs/style-guide/processes-procedures/understand-edit-docs-site-structure.mdx [hunk 7]
```diff
@@ -54,8 +54,8 @@ This has important implications, including:
 The nav files are quite simple. A nav file controls these things: 
 
 * The docs structure (the various levels of docs) for that category of docs. 
-* The category headers, set by `title` (e.g., "On-host integrations list") and `path` (e.g., "/docs/integrations/on-host-integrations"). 
-* The doc information, set by `title` (e.g., "NGINX integration") and `path` (the doc's URL).
+* The category headers, set by `title` (for example `On-host integrations list`) and `path` (for example `/docs/integrations/on-host-integrations`).
```
Comment (rhetoric101) [inline src/content/docs/style-guide/processes-procedures/understand-edit-docs-site-structure.mdx:57]: @austin-schaefer, how about adding a comma following "for example"?

File: src/content/docs/style-guide/processes-procedures/understand-edit-docs-site-structure.mdx [hunk 8]
```diff
@@ -54,8 +54,8 @@ This has important implications, including:
 The nav files are quite simple. A nav file controls these things: 
 
 * The docs structure (the various levels of docs) for that category of docs. 
-* The category headers, set by `title` (e.g., "On-host integrations list") and `path` (e.g., "/docs/integrations/on-host-integrations"). 
-* The doc information, set by `title` (e.g., "NGINX integration") and `path` (the doc's URL).
+* The category headers, set by `title` (for example `On-host integrations list`) and `path` (for example `/docs/integrations/on-host-integrations`). 
+* The doc information, set by `title` (for example `NGINX integration`) and `path` (the doc's URL).
```
Comment (rhetoric101) [inline src/content/docs/style-guide/processes-procedures/understand-edit-docs-site-structure.mdx:58]: @austin-schaefer, how about adding a comma following "for example"?

File: src/content/docs/style-guide/quick-reference/usage-dictionary.mdx [hunk 9]
```diff
@@ -323,9 +323,9 @@ For consistency across New Relic content, here are our general word usage guidel
     id="eg"
     title="e.g. and i.e."
   >
-    Don't use Latin abbreviations. 
+    Avoid these Latin abbreviations. 
     
-    Instead of `e.g.`, use `for example` or `such as`. Instead of `i.e.` or its English equivalent `in other words`, rewrite so your description is clear.
+    Instead of `e.g.`, use `for example` or `such as`. Subbing for `i.e.` is tricker, because sometimes people use it to mean `that is`, while at other times it's mis-used to mean `for example`. Read the context to be clear, but usually the best solution is to rewrite so the description is clear without needing explanatory text.
```
Comment (rhetoric101) [inline src/content/docs/style-guide/quick-reference/usage-dictionary.mdx:328]: @austin-schaefer, nice clarification!

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx [hunk 10]
```diff
@@ -9,7 +9,7 @@ Use this document to learn how to review and peer edit docs for you fellow write
 
 ## Peer editing workflow in GitHub [#peer-edit]
 
-If you’re peer editing a doc or have been otherwise assigned to a PR as a reviewer, you have a few choices for how and where to do the work.  The most streamlined and open-source approach is to do the edit using GitHub options, rather than copying the file to Google Docs and editing there. 
+If you’re peer editing a doc or have been otherwise assigned to a PR as a reviewer, you have a few choices for how and where to do the wor. The most streamlined and open-source approach is to do the edit using GitHub options, rather than copying the file to Google Docs and editing there.
```
Comment (rhetoric101) [inline src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:12]: @austin-schaefer, a "k" is missing from "work."

**Review by rhetoric101 — APPROVED** [review]
@austin-schaefer, these are great improvements!

I inserted some suggested changes. See what you think!

File: src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx (hunk 4 above)
Comment (austin-schaefer) [reply src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx:234]: Hey @rhetoric101 , not seeing a blank line in my editor?

image

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx (hunk 5 above)
Comment (austin-schaefer) [reply src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx:104]: For me, the line 72 one reads better without a comma?

File: src/content/docs/style-guide/quick-reference/usage-dictionary.mdx (hunk 9 above)
Comment (austin-schaefer) [reply src/content/docs/style-guide/quick-reference/usage-dictionary.mdx:328]: Thanks!

File: src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx (hunk 10 above)
Comment (austin-schaefer) [reply src/content/docs/style-guide/writer-workflow/peer-editor-workflow.mdx:12]: sheesh, nice catch.

**Pull request merged**